src/directives/CleanHtml/CleanHtml.js

```diff
@@ -1,5 +1,24 @@
 import DOMPurify from 'dompurify'
 
+/**
+ * As we allow the target attr, we have to make sure that the link is safe.
+ * Chromium seems to add `rel="noopener"` automatically as of v88, however
+ * to also ensure correct behavior for other engines, the `rel` attr is added
+ * whenever a target is set to "_blank".
+ *
+ * @see https://developer.chrome.com/docs/lighthouse/best-practices/external-anchors-use-rel-noopener/
+ */
+DOMPurify.addHook('afterSanitizeAttributes', function (node) {
+  const nodeHasTarget = 'target' in node
+  if (!nodeHasTarget) {
```

Comment:
why not 
```
if ('target' in node === false) {
```
?

Reply:
this way, ui suggests to "simplify the expression" which i did with b665fb76e4703c634fccd754d9b17dacaeacb640